hspec-core/src/Test/Hspec/Core/Runner.hs

```diff
@@ -309,3 +310,6 @@ randomizeForest :: Integer -> [Tree c a] -> [Tree c a]
 randomizeForest seed t = runST $ do
   ref <- newSTRef (mkStdGen $ fromIntegral seed)
   shuffleForest ref t
+
+countExamplesInSpec :: [EvalTree] -> Int
+countExamplesInSpec = getSum . foldMap (foldMap . const $ Sum 1)
```

Comment:
We use "example" in many places mostly for historic reasons.  For new code I prefer "spec item" instead.  So please rename to e.g. `countSpecItems` and `numberOfItems`.

Reply:
What do you prefer for doc comments: be consistent with nearby doc comments that use the term ‘examples’, or use the new ‘spec items’?

Reply:
"spec items".

We want to adjust existing comments that use "examples" for consistency (PRs welcome).